### PR TITLE
Added Munchausen syndrome

### PR DESCRIPTION
**Pull request by vchrombie**
Added Munchausen syndrome to the readme.md

Kindly notify if required any changes.
This solves the issue #11

### CODE REVIEW

**Review by icyflame — CHANGES_REQUESTED** [review]
Hey @vchrombie, thanks for your patch. Your references are good.

Can you please mention Munchausen by proxy in the paragraph about the entry? That form is more popularly known, through books and movies, and I would like a sentence or two about it here. You can remove the part about Munchausen not being connected to monetary or other benefits.

**Comment by vchrombie** [comment]
Added 'Munchausen syndrome by proxy' topic to the readme.md with an explanatory image also.
Can you please review it and tell me if I have to do any more changes.

**Review by icyflame — APPROVED** [review]

**Review by icyflame — CHANGES_REQUESTED** [review]
Hey, it seems like Munchausen by proxy is completely different. You can add another entry for that, and use separate references as well. That will be clearer. Thanks for your work!

**Comment by vchrombie** [comment]
I changed the subject about MBPS (Munchausen By Proxy Syndrome).
Can you review it please?

**Comment by vchrombie** [comment]
Can you review this PR asap please.

I cannot place a new PR for adding Placebo effect #17

**Review by icyflame — CHANGES_REQUESTED** [review]
Add two separate entries for MS and MSBP.

**Comment by vchrombie** [comment]
Okay I will do it.
Thank You.

**Comment by vchrombie** [comment]
I have divided MS and MSBP.
Can you review it now?

**Comment by vchrombie** [comment]
@RonLek @icyflame  Anyone, can you please review this PR.

File: readme.md [hunk 1]
```diff
@@ -177,6 +179,31 @@ the anecdote, you can throw away the data."
 [R: verywell.com](https://www.verywell.com/what-is-the-hawthorne-effect-2795234)
 
 
+#### [Munchausen syndrome](https://en.wikipedia.org/wiki/Factitious_disorder_imposed_on_self)
+
+> Person repeatedly and deliberately acts as if he or she has a physical or mental illness when he or she is not really sick.
```
Comment (icyflame) [inline readme.md]: Shorten this sentence

File: readme.md [hunk 2]
```diff
@@ -177,6 +179,31 @@ the anecdote, you can throw away the data."
...
+
+![Image](./img/Munchausen_syndrome.png)
+
+Munchausen's syndrome is a psychological and behavioural condition where someone pretends to be ill or induces symptoms of illness in themselves. It is also sometimes known as factitious disorder. People with the condition intentionally produce or pretend to have physical or psychological symptoms of illness. Their main intention is to assume the "sick role" to have people care for them and be the centre of attention.
```
Comment (icyflame) [inline readme.md]: Wrap at 70

File: readme.md [hunk 3]
```diff
@@ -177,6 +179,31 @@ the anecdote, you can throw away the data."
...
+Munchausen's syndrome is a psychological and behavioural condition where someone pretends to be ill or induces symptoms of illness in themselves. It is also sometimes known as factitious disorder. People with the condition intentionally produce or pretend to have physical or psychological symptoms of illness. Their main intention is to assume the "sick role" to have people care for them and be the centre of attention. 
+
+[R: nhs.uk](https://www.nhs.uk/conditions/munchausens-syndrome/)
+
+[R: my.clevelandclinic.org](https://my.clevelandclinic.org/health/articles/munchausen-syndrome)
+
+
+#### [Munchausen syndrome by proxy](https://en.wikipedia.org/wiki/Factitious_disorder_imposed_on_another)
+
+> Person acts as if an individual he or she is caring for has a physical or mental illness when the person is not really sick.
```
Comment (icyflame) [inline readme.md]: Shorten

File: readme.md [hunk 4]
```diff
@@ -177,6 +179,31 @@ the anecdote, you can throw away the data."
...
+
+![Image](./img/Munchausen_syndrome-by-proxy.png)
+
+Munchausen by proxy syndrome is a relatively rare form of child abuse that involves the exaggeration or fabrication of illnesses or symptoms by a primary caretaker. The adult perpetrator lies about illness in another person under his or her care, usually a child under 6 years of age. "By proxy" indicates that a parent or other adult is making up or exaggerating symptoms in a child, not in himself or herself.
```
Comment (icyflame) [inline readme.md]: Wrap at 70

File: readme.md [hunk 5]
```diff
@@ -177,6 +179,31 @@ the anecdote, you can throw away the data."
...
+
+[R: .healthline.com](https://www.healthline.com/health/munchausen-syndrome-by-proxy)
+
+[R: .medicinet.com](https://www.medicinenet.com/script/main/art.asp?articlekey=38327)
```
Comment (icyflame) [inline readme.md]: This ref doesn't cut it. Please find a better ref. :slightly_smiling_face:

**Review by icyflame — CHANGES_REQUESTED** [review]

File: readme.md (hunk 1 above)
Comment (vchrombie) [reply readme.md]: Okay. :+1:

File: readme.md (hunk 3 above)
Comment (vchrombie) [reply readme.md]: Okay. :+1:

File: readme.md (hunk 5 above)
Comment (vchrombie) [reply readme.md]: Okay. :+1:

File: readme.md (hunk 2 above)
Comment (vchrombie) [reply readme.md]: Sorry, I didn't get you.
Wrap at 70?

Should I shorten the paragraph?

Comment (vchrombie) [reply readme.md]: I have shortened it.

File: readme.md (hunk 4 above)
Comment (vchrombie) [reply readme.md]: I did it.

**Comment by vchrombie** [comment]
Is it good now?
Can you please review it.

**Comment by icyflame** [comment]
Merged your changes into master.

Despite repeated requests to wrap the paragraph at 70 characters, you failed to do it.

I have merged it in because I don't have the time to keep reviewing this again and again, but you need to up your game when it comes to _reading reviews from maintainers and following them_.

**Pull request closed without merging**